.github/workflows/release.yml

```diff
@@ -36,19 +36,36 @@ jobs:
       - name: Install NPM packages
         run: cd ui && rm yarn.lock && yarn install
 
+      - name: Build UI Bundle
+        run: cd ui && yarn build
+
+      - name: Create Release Archive
+        run: tar -czvf ui-assets.tar.gz -C ui/build .
+
       - name: Build release binary
         run: |
           GOOS=${{ matrix.goos }} GOARCH=amd64 make build
           tar -czvf asynqmon_${{ steps.release.outputs.tag_name }}_${{ matrix.goos }}_amd64.tar.gz asynqmon
           ls
 
-      - name: Upload Release Asset
-        id: upload-release-asset
+      - name: Upload UI Release Asset
```

Comment:
> Were you able to verify this updated Github workflow works?

No, I don't know how to simulate that but I think it should work. We can try with a test release maybe.

The idea is to still publish a release binary, but also publish a bundle of UI (which can be used to embed into other go binaries), so that folks can download it from GitHub Releases and they won't have to make a yarn build for using it.